routes/api.js

```diff
@@ -19,6 +19,7 @@ const router = express.Router();
 // Auth
 router.post('/user/register', AuthController.register);
 router.post('/user/login', AuthController.login);
+router.post('/user/password/reset', ProtectMiddleware, AuthController.resetPassword);
```

Comment:
maybe we use a PATCH request. 
```suggestion
router.patch('/user/reset/password', ProtectMiddleware, AuthController.resetPassword);
```